src/spec/angular-material-mocks.js

```diff
@@ -0,0 +1,77 @@
+/**
+ *
+ * Angular-Material-Mocks
+ *
+ * Developers interested in running their own custom unit tests WITH angular-material.js loaded...
+ * must also include this *mocks* file. Similar to `angular-mocks.js`, `angular-material-mocks.js`
+ * will override and disable specific Angular Material performance settings:
+ *
+ *  - Disabled Theme CSS rule generations
+ *  - Forces $mdAria.expectWithText() to be synchronous
+ *  - Mocks $$rAF.throttle()
+ *  - Captures flush exceptions from $$rAF
+ *
+ */
+(function(window, angular, undefined) {
```

Comment:
The whole "passing in `undefined` / `window`" thing isn't really necessary anymore (ES5), and is actually more prone to error that just using the global (which is now spec'ed to not be overwritable).